### PR TITLE
MAT-9067 admin api throws 500

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## MADiE PR

Jira Ticket: [MAT-9067](https://jira.cms.gov/browse/MAT-9067)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/java/cms/gov/madie/measure/resources/AdminController.java [hunk 1]
```diff
@@ -472,7 +472,7 @@ private boolean isDirectAncestor(Measure sourceMeasure, Measure targetMeasure) {
     if (!StringUtils.equals(
         sourceMeasure.getMeasureSet().getMeasureSetId(),
         targetMeasure.getMeasureSet().getMeasureSetId())) {
-      throw new InvalidRequestException("Source measure from different Measure family/set.");
+      return false;
```
Comment (jkotanchik-SB) [inline src/main/java/cms/gov/madie/measure/resources/AdminController.java]: This can be simplified to a single statement now that we're not throwing the exception.
```suggestion
return StringUtils.equals(
sourceMeasure.getMeasureSet().getMeasureSetId(),
targetMeasure.getMeasureSet().getMeasureSetId()));
```

Comment (sb-cecilialiu) [reply src/main/java/cms/gov/madie/measure/resources/AdminController.java]: @jkotanchik-SB Good suggestion, I combined the two separate comparison into one. Thanks Joe!

**Review by jkotanchik-SB — APPROVED** [review]
